jobfunnel/config/search.py

```diff
@@ -0,0 +1,79 @@
+"""Object to contain job query metadata
+"""
+from typing import List, Optional
+from jobfunnel.config import BaseConfig
+from jobfunnel.resources import Locale, Provider
+from jobfunnel.resources.defaults import (
+    DEFAULT_SEARCH_RADIUS_KM, DEFAULT_MAX_LISTING_DAYS,
+    DEFAULT_DOMAIN_FROM_LOCALE,
+)
+
+class SearchConfig(BaseConfig):
```

Comment:
Really like how the configurations are encapsulated. Will make it easier to add more scrapers in the future :+1:.